packages/components/app/styles/components/app-footer.scss

```
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+$app-footer-gap: 24px;
+$app-footer-icon-text-gap: 6px;
+
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $app-footer-gap;
+  justify-content: flex-end;
+  padding: 24px;
+  color: var(--app-footer-foreground-color);
+  border-top: 1px solid var(--app-footer-border-top-color);
+}
+
+// SUB-ELEMENTS
+
+// hide list if no children
+.hds-app-footer__list:not(:has(li)) { display: none; }
+
+.hds-app-footer__list,
+.hds-app-footer__legal-links {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $app-footer-gap;
+  align-items: center;
+  justify-content: flex-end;
+  width: max-content;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.hds-app-footer__status-link {
+  // custom spacing for the status link (internally is using the "AppFooter::Link")
+  // Note: we increase specificity because otherwise is overwritten by the order of imports of components scss files
+  &.hds-link-inline--icon-leading > .hds-link-inline__icon { margin-right: $app-footer-icon-text-gap; }
+
+  .flight-icon {
+    fill: var(--hds-app-footer-status-icon-color, currentColor);
+  }
+}
+
+// preset status types
+.hds-app-footer__status-link--operational {
+  .flight-icon { fill: var(--app-footer-status-link-icon-operational-color); }
+}
+
+.hds-app-footer__status-link--degraded {
+  .flight-icon { fill: var(--app-footer-status-link-icon-degraded-color); }
+}
+
+.hds-app-footer__status-link--maintenance {
+  .flight-icon { fill: var(--app-footer-status-link-icon-maintenance-color); }
+}
+
+.hds-app-footer__status-link--critical {
+  .flight-icon { fill: var(--app-footer-status-link-icon-critical-color); }
+}
+
+
+.hds-app-footer__link.hds-link-inline--color-secondary,
+.hds-app-footer__status-link {
+  // Overriding default colors
+  color: var(--app-footer-link-default-color);
```

extra safety in case of long strings that wrap multiple lines
```suggestion
  color: var(--app-footer-link-default-color);
  text-align: right;
```